src/Data/List/Relation/Unary/AllPairs.agda

```diff
@@ -23,65 +26,54 @@ import Relation.Nullary.Decidable as Dec
 ------------------------------------------------------------------------
 -- Definition
 
--- AllPairs R xs means that every pair of elements (x , y) in xs is a
--- member of relation R (as long as x comes before y in the list).
-
-infixr 5 _∷_
-
-data AllPairs {a ℓ} {A : Set a} (R : Rel A ℓ) : List A → Set (a ⊔ ℓ) where
-  []  : AllPairs R []
-  _∷_ : ∀ {x xs} → All (R x) xs → AllPairs R xs → AllPairs R (x ∷ xs)
+open import Data.List.Relation.Unary.AllPairs.Core public
```

Comment:
This unqualified re-export in a `{a ℓ} {A : Set a} {R : Rel A ℓ}` context looks a bit dodgy though.